src/components/dialogs/field-hook.js

```diff
@@ -83,61 +80,6 @@ export const useTextValue = ({
     return [value, field, setValue, hasChanged];
 };
 
-export const useFileValue = ({ fileExceedsLimitMessage, isLoading }) => {
-    const selectedFile = useSelector((state) => state.selectedFile);
-    const intl = useIntl();
-    const dispatch = useDispatch();
-    const [fileOk, setFileOk] = useState(false);
-    const [fileError, setFileError] = useState();
-
-    const field = <UploadCase isLoading={isLoading} />;
-
-    const resetSelectedFile = useCallback(
-        () => dispatch(removeSelectedFile()),
-        [dispatch]
-    );
-
-    useEffect(() => {
-        const MAX_FILE_SIZE_IN_MO = 100;
-        const MAX_FILE_SIZE_IN_BYTES = MAX_FILE_SIZE_IN_MO * 1024 * 1024;
-        if (!selectedFile) {
-            setFileError();
-            setFileOk(false);
-        } else if (selectedFile.size <= MAX_FILE_SIZE_IN_BYTES) {
-            setFileError();
-            setFileOk(true);
-        } else {
-            setFileError(
-                fileExceedsLimitMessage
-                    ? fileExceedsLimitMessage
-                    : intl.formatMessage(
-                          {
-                              id: 'uploadFileExceedingLimitSizeErrorMsg',
-                          },
-                          {
-                              maxSize: MAX_FILE_SIZE_IN_MO,
-                              br: <br />,
-                          }
-                      )
-            );
-            setFileOk(false);
-        }
-    }, [selectedFile, fileExceedsLimitMessage, intl]);
-    return [
-        selectedFile,
-        field,
-        fileError,
-        fileOk,
-        resetSelectedFile,
-        setFileOk,
-    ];
-};
-
-const makeAdornmentEndIcon = (content) => {
-    return {
-        endAdornment: <InputAdornment position="end">{content}</InputAdornment>,
-    };
-};
 export const useNameField = ({
```

Comment:
We will remove this in an other PR maybe ?

Reply:
I think that is the scope of the TS to refactor this custom hooks

Reply:
So why we still use it ?
The point of migrating to RHF is to use RHF inputs and not theses custom hooks

Reply:
I mean, there is an other TS to refactor CreateDirectoryDialog ? If no, we should create one

Reply:
ok thank you